src/features/layout/Layout.tsx

```diff
@@ -7,9 +7,11 @@ const Layout = () => {
   return (
     <>
       <Nav />
-      <div>
-        <Outlet />
-      </div>
+      <main className=" mx-auto flex h-[80vh] flex-col xl:max-w-7xl ">
```

Comment:
Maybe we dont need xl: to set max-w ?

Reply:
it's not pushed yet, but we changed it on call :)